subprojects/logging/src/main/java/org/gradle/internal/logging/LoggingManagerInternal.java

```diff
@@ -50,4 +50,9 @@
     LoggingManagerInternal captureStandardError(LogLevel level);
 
     LoggingManagerInternal setLevelInternal(LogLevel logLevel);
+
+    /**
+     * Sets the worker count to display in the UI.
+     */
+    LoggingManagerInternal setMaxWorkerCount(int maxWorkerCount);
```

Comment:
I'm unsure of calling this feature _max worker count_ as it makes less sense when it comes to logging. I was thinking dropping _max_ and simply have _worker count_.